README.md

```diff
@@ -13,6 +13,51 @@ These witnessed checkpoints can be consumed by clients that want protection agai
 
 Users wishing to run this should start with the [OmniWitness](./cmd/omniwitness/).
 
+## Importance of Witnesses
+
+Some important terminology:
+
+Term                | Definition
+--------------------|------------
+Log                 | A verifiable data structure that commits to a list of entries
+Append Only         | New entries added to a log can only be added to the end of the list
+Checkpoint          | A Checkpoint is a _signed_ commitment to the state of a Log at a given size
+Split View          | A split view is a situation where a Log presents two views of its data where one is not a prefix of the other
+Consistency Proof   | A crytographic proof between two checkpoints that proves one of them is an Append Only evolution of the other
+Witness             | Tracks one or more logs, verifying Consistency Proofs to ensure it cannot cross any Split Views
```

Comment:
I think it might be important to add here that a Witness also attests to consistency.

Reply:
Added that it signs checkpoints.